client/graphics/lights.hpp

```diff
@@ -13,23 +13,17 @@
 
 namespace tec {
 	struct BaseLight {
-		glm::vec3 color;
-		float ambient_intensity;
-		float diffuse_intensity;
-
-		BaseLight() {
-			color = glm::vec3(1.0f, 1.0f, 1.0f);
-			ambient_intensity = 0.5f;
-			diffuse_intensity = 0.2f;
-		}
+		glm::vec3 color{ 1.f, 1.f, 1.f };
+		float ambient_intensity{ 0.5f };
+		float diffuse_intensity{ 0.2f };
+
+		BaseLight() = default;
```

Comment:
Wow that's much better than before. 

I would prefer copy-initialization or copy-list-initialization for readability though.